website/docs/r/cloudaccount_aws.html.markdown

```diff
@@ -111,6 +111,31 @@ resource "dome9_cloudaccount_AWS" "test" {
 }
 ```
 
+```hcl
+resource "dome9_cloudaccount_AWSGOV" "test" {
```

Comment:
there is no resource dome9_cloudaccount_AWSGOV but dome9_cloudaccount_aws with vendor awsgov